src/tests/Quote.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Quotes from '../components/Quote';
+
+it('matches snapshot', () => {
+  const tree = renderer.create(
+    <Quotes />,
+  ).toJSON();
+  expect(tree).toMatchSnapshot();
```

Comment:
- [ ] One of your test snapshots is failing. Kindly check it out

![image](https://user-images.githubusercontent.com/98453979/232219619-d6158943-d2f5-4d9e-9f4e-da3ffbf50203.png)